fe/fe-core/src/main/java/com/starrocks/load/InsertOverwriteJobManager.java

```diff
@@ -0,0 +1,239 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.load;
+
+import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
+import com.google.gson.annotations.SerializedName;
+import com.starrocks.catalog.Database;
+import com.starrocks.catalog.OlapTable;
+import com.starrocks.common.io.Text;
+import com.starrocks.common.io.Writable;
+import com.starrocks.persist.CreateInsertOverwriteJobInfo;
+import com.starrocks.persist.InsertOverwriteStateChangeInfo;
+import com.starrocks.persist.gson.GsonPostProcessable;
+import com.starrocks.persist.gson.GsonUtils;
+import com.starrocks.qe.ConnectContext;
+import com.starrocks.qe.StmtExecutor;
+import com.starrocks.server.GlobalStateMgr;
+import com.starrocks.sql.common.MetaUtils;
+import io.netty.util.concurrent.DefaultThreadFactory;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ThreadFactory;
+import java.util.concurrent.locks.ReentrantReadWriteLock;
+
+public class InsertOverwriteJobManager implements Writable, GsonPostProcessable {
+    private static final Logger LOG = LogManager.getLogger(InsertOverwriteJobManager.class);
+
+    @SerializedName(value = "overwriteJobMap")
+    private Map<Long, InsertOverwriteJob> overwriteJobMap;
+
+    // tableId -> partitionId list
+    @SerializedName(value = "partitionsWithOverwrite")
+    private Map<Long, List<Long>> partitionsWithOverwrite;
```

Comment:
Can you add more comments for this object?

Reply:
it is modified to tableToOverwriteJobs